osbuild/__init__.py

```diff
@@ -16,3 +16,10 @@
     "Pipeline",
     "Stage",
 ]
+
+try:
+    from pkg_resources import get_distribution
+
+    __version__ = get_distribution("osbuild").version
```

Comment:
This probably does not work for non-installed `osbuild`, i.e. when running from the source dir.
Corner case: osbuild is installed but we are still running from the source dir. I wonder if we should care about this case.

Reply:
Yea, in fact the exception of the distribution not being found should be caught and handled. The corner case needs some thought.